testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveCompression.java

```diff
@@ -83,6 +83,34 @@ public void testSnappyCompressedParquetTableCreatedInHive()
         onHive().executeQuery("DROP TABLE " + tableName);
     }
 
+    @Test(groups = HIVE_COMPRESSION)
+    public void testSnappyCompressedLargeCreateTableAsInTrino()
```

Comment:
For https://github.com/trinodb/trino/issues/5518 i assume compression is irrelevant, but "large table" is.

- remove Snappy from method name and code
- add a javadoc that this is a regression test for https://github.com/trinodb/trino/issues/5518

Reply:
if compression is irrelevant, the test wouldn't belong in this class

BTW worth adding similar one for Iceberg?

Reply:
I was thinking of putting it in a more general test group but since it's a large table I didn't want to run it too many times. Same for duplicating the test for iceberg.

Reply:
This can go into storage format class (or separate class). But shouldn't be in storage_formats group (add a why, unless separate class)